.github/workflows/tryrebase.yml

```diff
@@ -38,3 +39,13 @@ jobs:
           else
             python3 .github/scripts/tryrebase.py "${PR_NUM}"
           fi
+      - name: Comment on Canceled
+        if: ${{ cancelled() && steps.checkout.outcome == 'success' }}
```

Comment:
Is the only reason we need checkout to be a success just so we could access the comment_on_pr file?

Reply:
Yea. If the checkout isn't there, it's gonna call this script and it's gonna fail since the script doesn't exist yet.

Reply:
I was thinking (this can be a followup) that we can just have the script be in the GHA as a shell command in the run section since there is a python shell.